webview/src/experiments/components/emptyState/AddColumns.tsx

```diff
@@ -1,10 +1,28 @@
 import React from 'react'
+import { useSelector } from 'react-redux'
+import { selectColumns, toggleShowOnlyChanged } from '../../util/messages'
+import { ExperimentsState } from '../../store'
 import { StartButton } from '../../../shared/components/button/StartButton'
-import { selectColumns } from '../../util/messages'
+import { IconButton } from '../../../shared/components/button/IconButton'
+import { Close } from '../../../shared/components/icons'
 
-export const AddColumns: React.FC = () => (
-  <div>
-    <p>No Columns Selected.</p>
-    <StartButton onClick={selectColumns} text="Add Columns" />
-  </div>
-)
+export const AddColumns: React.FC = () => {
+  const showOnlyChanged = useSelector(
+    (state: ExperimentsState) => state.tableData.showOnlyChanged
+  )
+  return (
+    <div>
+      <p>No Columns Selected.</p>
+      <StartButton onClick={selectColumns} text="Add Columns" />
+      {showOnlyChanged && (
```

Comment:
[F] Good thing I remembered this because without it users would have been stuck.